### PR TITLE
Add specific error for invalid operator in VRs

### LINKED ISSUE #9: Add check and specific error for unknown operator
Add check in visual rules for unknown operators , returning an specific error

**Effort: 2 man day**

### PR DESCRIPTION
**Pull request by crbrox**
Specific error for unknown operators in VisualRules 
Close #9

@fgalan 
@dmoranj 
@jcalderin

### CODE REVIEW

**Comment by jcalderin** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete branch)

File: lib/models/visualRules.js [hunk 1]
```diff
@@ -38,6 +38,13 @@ var util = require('util'),
     },
     errors = {};
 
+function errorOperator(op) {
```
Comment (fgalan) [inline lib/models/visualRules.js]: Sorry for this post-LGTM comment, but I have just realized...

Do we need entry at CHANGES_NEXT_RELESE for these changes? Probably not, due that are small and internal ones, but just to let you know...

Comment (crbrox) [inline lib/models/visualRules.js]: After talk, NTC

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**